CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ CHANGELOG
   * `Razorpay`
   * `Systempay`
 
```

Comment:
There is an extra newline here, but none of the previous lists had newlines between items.

Reply:
Ah my bad.  I've force pushed the change